src/components/Toggle-button/style.scss

```diff
@@ -74,8 +81,7 @@
 
 
     .toggle-button__button {
-        display: inline-block;
-        margin-top: 5px;
+        margin-top: -1px;
```

Comment:
are those changes inline with this ticket - https://github.com/kaltura/express-recorder/pull/178?

Reply:
yes, the margin-top of the toggle button does not affect either the full view, nor 400 zoom view